platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/PreconsensusEventFile.java

```diff
@@ -56,7 +56,7 @@
 public final class PreconsensusEventFile implements Comparable<PreconsensusEventFile> {
 
     /**
-     * The file extension for standard files. Stands for "Pre-Consensus Event Stream".
+     * The file extension for standard files. Stands for "PreConsensus Event Stream".
```

Comment:
"PreConsensus"

Reply:
This one is actually intentional, since I'm explaining an acronym.